PROTOCOL.md

```diff
@@ -306,8 +306,40 @@ The device should answer:
          },
      "id" : <same number>
 }
+```
+
+#### Controller wants the device to apply a given fixed configuration
+
+Controller sends this command when it requires the device to apply fixed condfiguration, eg. country code. The device
```

Comment:
check spelling: condfiguration
also change to: ... applied and is running.

Reply:
Changed wording a bit to be in-sync with the code.